StarVMC/StarAgmlLib/AgModule.cxx

```diff
@@ -45,57 +45,14 @@ Bool_t AgModule::AddHit( string _for, string meas, Float_t bitns, Float_t mn, Fl
 
 Bool_t AgModule::AddCut( string block, string cut, Float_t value )
 {
-
-  TDataSet      *BLOCKS   = mDataSet -> Find("blocks");
-  TDataSet      *THEBLOCK = BLOCKS   -> Find(block.c_str());
-  TGenericTable *G3cuts   = dynamic_cast<TGenericTable *>( THEBLOCK -> Find("G3cuts") );
-
-
-  if ( 0==G3cuts )
-    {
-      //      cout << "Register new cut table for block " << block.c_str() << endl;
-      // Create the table and add default set of cuts
-      THEBLOCK -> Add( G3cuts = new TGenericTable( "Gccuts_t", "G3cuts" ) );
-      Gccuts_t cuts; /* new default cuts */ 
-      G3cuts -> AddAt( &cuts );
-    }
-
-  // Set new cut value for this cut
-  ((Gccuts_t *)G3cuts->At(0))->set( cut, value );
```

Comment:
Why not nuke the function altogether, or, at least, make it call `std::abort()`?

Reply:
Ok